app/src/main/res/values/styles_textview.xml

```diff
@@ -27,4 +27,10 @@
         <item name="android:paddingEnd">@dimen/create_account_description_padding</item>
     </style>
 
+    <style name="TextView.Authentication.Header" parent="TextView.H2">
+        <item name="android:textColor">?android:textColorPrimary</item>
+        <item name="android:gravity">center_horizontal</item>
+        <item name="android:textSize">32sp</item>
```

Comment:
Is this header text size unique? None of the parent header styles H1 through to H6 cover this text size? I'm dubious about us having these concrete parent styles and overriding text sizes unless this is completely unique? I feel it'll take away from the consistency from a design perspective. Obviously, it's not easy when we are using old designs, so i'm not too concerned here.

Reply:
there's no pre-defined style that gives us this font + size + color combination. We have 34 dp in some of them, but then it has a different font & color. TextView.H2 was the one that I can extend with minimal changes. Unfortunately this doesn't resemble any other predefined styles, so I couldn't use my self-judgement here to change it :) instead I kept what's there on old design.

Reply:
That's completely fair, that is one of the tradeoffs with replicating with an old design, i'll resolve this for now